docs/en/Running-Hadoop-MapReduce-on-Alluxio.md

```diff
@@ -28,13 +28,13 @@ the `hdfs-site.xml` file as well.
 
 ## Using Hadoop 2.x
 
-If you are using a 2.x Hadoop cluster, you should not need the properties above in your
-`core-site.xml` file. However, in some cases you may encounter the error:
+If you are using a Hadoop 2.x cluster, you need to configure `alluxio.hadoop.AlluxioFs` as Hadoop AbstractFileSystem. 
+The properties of Hadoop 1.x is optional. However, in some cases you may encounter the error:
 `java.io.IOException: No FileSystem for scheme: alluxio`. For instance, this may happen when Yarn 
 (as opposed to Hadoop) tries to access Alluxio files. If this error is encountered, add these 
 properties to your `core-site.xml` file, and restart Yarn.
 
-{% include Running-Hadoop-MapReduce-on-Alluxio/config-core-site.md %}
+{% include Running-Hadoop-MapReduce-on-Alluxio/config-core-site-hadoop-2.x.md %}
```

Comment:
Yes. This is a good idea! There is no downside for adding this property. Basically, Hadoop 2.x uses two mixed interfaces together. e.g. in Hadoop dfs command line still uses the old FileSystem interface, and Yarn MRv2 requires AbstractFileSystem interface. So we can just one fixed configuration and share it to Hadoop 1.x and Hadoop 2.x.